crates/cxx-qt-lib/src/core/qsizef.rs

```diff
@@ -42,6 +42,18 @@ mod ffi {
         #[doc(hidden)]
         #[rust_name = "qsizef_to_qstring"]
         fn toQString(value: &QSizeF) -> QString;
+        #[doc(hidden)]
+        #[rust_name = "qsizef_plus"]
+        fn operatorPlus(a: &QSizeF, b: &QSizeF) -> QSizeF;
+        #[doc(hidden)]
+        #[rust_name = "qsizef_minus"]
+        fn operatorMinus(a: &QSizeF, b: &QSizeF) -> QSizeF;
+        #[doc(hidden)]
+        #[rust_name = "qsizef_mul"]
+        fn operatorMul(a: f32, b: &QSizeF) -> QSizeF;
```

Comment:
why is this `f32` when `QSize` is `f64` and it's `qreal` for both in C++ ?

Reply:
Oops, I missed it.